### DISCUSSION TITLE
Block styles in inline mode (displayMode: false)

### DISCUSSION THREAD

**Discussion by alandouglas96** [KaTeX/KaTeX#2631] [Q&A] [answered]
Is it possible to have inline formulas displayed with the styles used in block mode?

We would love to have that option in for example, displaying limits `\lim_{x \to 0}`.

Thanks for this amazing project from Edpuzzle!

**Comment by edemaine** [comment, marked as answer]
You can use `\displaystyle` to switch to the style of displayed formulas within inline mode, e.g. `\displaystyle \lim_{x \to 0}`. Is this sufficient, or do you want to change *all* formulas to `\displaystyle`?

**Reply by alandouglas96** [reply]
Thanks for the rapid response. This will do for us! But we are curious, is the second option you mention possible?

**Reply by edemaine** [reply]
It depends how you're using KaTeX. If you're using autorender, you could use the `preProcess` callback to automatically prefix `\displaystyle` to every math formula. See https://katex.org/docs/autorender.html

**Reply by alandouglas96** [reply]
Thanks again for the suggestion! I am afraid we are not able to use autorender, but we can work it out with your first suggestion. 👌
